day21/helpingStephan.py

```diff
@@ -0,0 +1,40 @@
+def helpingStephan(x, y):
+    xSplit = str(x).split(".")
+    ySplit = str(y).split(".")
+    
+    XdigitsAfterDecimal = "0"
+    if len(xSplit) > 1:
+        XdigitsAfterDecimal = xSplit[1]
+        
+    YdigitsAfterDecimal = "0"
+    if len(ySplit) > 1:
+        YdigitsAfterDecimal = ySplit[1]
+                                      
+                                  
+    digitsAfterDecimal = max(len(XdigitsAfterDecimal), len(YdigitsAfterDecimal))
```

Comment:
instead of doing all this work, why not use the problem description where it says "It is guaranteed that both x and y will have at most 5 digits after the decimal point."